api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -112,6 +112,9 @@ message RBAC {
   // Maps from policy name to policy. A match occurs when at least one policy matches the request.
   // The policies are evaluated in lexicographic order of the policy name.
   map<string, Policy> policies = 2;
+
+  // [#extension-category: envoy.rbac.custom_library_config]
```

Comment:
Regarding naming, I prefer not to use "library" here, it's a bit too generic. "Custom attributes" is what I think is being added.